podcasts/Kids Profile/KidsProfileBannerView.swift

```diff
@@ -0,0 +1,108 @@
+import SwiftUI
+
+struct KidsProfileBannerView: View {
+    @EnvironmentObject var theme: Theme
+
+    let viewModel: KidsProfileBannerViewModel
+
+    var body: some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 8)
+                .fill(theme.primaryUi01Active)
+
+            HStack {
+                textsView
+                    .padding(.leading, 16)
+                    .padding(.trailing, 35)
+
+                Spacer()
+
+                ZStack(alignment: .topTrailing) {
+                    VStack {
+                        Spacer()
+
+                        Image("kids-profile-banner-face")
+                            .clipShape(RoundedRectangle(cornerRadius: 8))
+                    }
+
+                    VStack {
+                        Button(action: viewModel.closeButtonTap) {
+                            Image(systemName: "xmark")
+                                .imageScale(.small)
+                                .foregroundColor(theme.secondaryIcon02)
+                        }
+                        .frame(width: 24, height: 24)
+                        .padding(.top, 8)
+                        .padding(.trailing, 4)
+
+                        Spacer()
+                    }
+                }
+            }
+        }
+        .frame(height: 105)
+    }
+
+    var textsView: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(L10n.kidsProfileBannerTitle)
+                    .font(
+                        Font.custom("SF Pro Text", size: 15.0)
+                            .weight(.semibold)
+                    )
+                    .frame(alignment: .topLeading)
+                    .foregroundStyle(theme.primaryText01)
+
+                Text(L10n.kidsProfileBannerBadge)
+                    .font(
+                        Font.custom("SF Pro Text", size: 11)
+                            .weight(.semibold)
+                    )
+                    .foregroundColor(theme.secondaryText02)
+                    .opacity(0.8)
+                    .padding(.horizontal, 6)
+                    .padding(.vertical, 2)
+                    .background(theme.primaryUi05)
+                    .cornerRadius(4)
+            }
+
+            Text(L10n.kidsProfileBannerText)
+                .font(
+                    Font.custom("SF Pro Text", size: 11)
```

Comment:
Should I use a different font? As I don't see this one used as custom font in other places, but it's used in our design.
CC @leandroalonso

Reply:
Did you confirm with @david-gonzalez-a8c that this is intended?

Reply:
Nevermind, I found out. I checked the Give Rating project in Figma and the same font is used, while it's rendered by our font modifier in our code. I will replace it.